source/hardware/controllers/native_instruments_traktor_kontrol_s4_mk3.rst

```diff
@@ -369,29 +385,33 @@ Mapping options
+Whether or not to use haptic feedback features                                                 `UseMotors`                                 false                                                                                                             true/false                                                                            Whether or not to use haptic feedback features. This is a beta feature, some of them may be unstable.
+Map the mixer master knob the Mixxx internal mixer                                             `SoftwareMixerMain`                         false                                                                                                             true/false                                                                            When enable, the master potentiometer on top right column of the mixer will drive the main gain of the Mixxx internal mixer as well as the hardware built-in mixer in the device.
+Map the mixer booth knob the Mixxx internal mixer                                              `SoftwareMixerBooth`                        false                                                                                                             true/false                                                                            When enable, the booth potentiometer on right column of the mixer will drive the booth gain of the Mixxx internal mixer as well as the hardware built-in mixer in the device.
+Map the mixer booth knobs (cue and volume) the Mixxx internal mixer                            `SoftwareMixerHeadphone`                    false                                                                                                             true/false                                                                            When enable, the headphone volume and cue potentiometers on bottom right column of the mixer will drive the headphone controls of the Mixxx internal mixer as well as the hardware built-in mixer in the device.
+Default Pad Layout                                                                             `DefaultPadLayout`                          default                                                                                                           default,hotcue,samplerBeatloop,keyboard                                               Define the default layout used for the pads.
+============================================================================================== =========================================== ================================================================================================================= ===================================================================================== ================================================================================================================================================================================================================
```

Comment:
no idea what changed here. Just the column widths?

Reply:
Yep, I had to resize the description column to make the last 4 added items (`SoftwareMixerMain`...`DefaultPadLayout`) fit